src/components/organisms/deployments/sessions/viewer.tsx

```diff
@@ -268,6 +273,11 @@ export const SessionViewer = () => {
 						</Tab>
 					))}
 				</div>
+				{loadingOutputs || loadingActivities ? (
+					<div>
+						<Loader />
```

Comment:
1.:
```suggestion
						<Loader size="sm" />
```

2. Make sure it is centered vertically